Makefile

```diff
@@ -48,110 +56,110 @@ image-cleanup:
 
 service-start-highstack-as-ps-dclinks:
 	cd services; \
-	sed -i "s/fac-iocs:.*/fac-iocs:$(FAC_IOCS_TAG)/g" docker-stack-as-ps-dclinks.yml; \
-	docker stack deploy -c docker-stack-as-ps-dclinks.yml fac-as-ps-dclinks; \
+	sed -i "s/fac-iocs:.*/fac-iocs:$(FAC_IMG_IOCS_TAG)/g" docker-stack-as-ps-dclinks.yml; \
+	docker stack deploy -c docker-stack-as-ps-dclinks\.yml facs-as-ps-dclinks; \
```

Comment:
is the `\` in `docker-stack-as-ps-dclinks\.yml` on purpose?

Reply:
a replace mistake. fixed in another branch. doing it here too..